app/static/js/index.js

```diff
@@ -347,3 +347,24 @@ window.onscroll = function(e) {
         pagesFinished = true;
     }
 }
+
+// Scroll to top button
+var scrollTopBtn = document.getElementById('scrollToTopBtn')
+var rootElement = document.documentElement
+
+function showButton() {
+if ((rootElement.scrollTop / (rootElement.scrollHeight - rootElement.clientHeight)) > 0.15 ) {
+    scrollTopBtn.classList.add('showBtn')
+} else {
+    scrollTopBtn.classList.remove('showBtn')
```

Comment:
Rather than an if/else, you can use `scrollTopBtn.classList.toggle('shown', boolean_condition`